posts/homeless_hack_day.md

```diff
@@ -0,0 +1,63 @@
+# Homeless Hackday London
+
+At Made Tech, we run an internal Hack Day or Learning Day every month. We take time off our paid work to focus on learning something new (like a new programming language) or to hack together some tools that solve a problem we're encountering. 
+
+Some of our recent hack days have covered:
+
+* Learning Kotlin
+* Uncle Bob's Clean Architecture
+* Creating a Smart Office
+* Hardening Security
+* Making Computer Games
+* Creating Dashboards / Information Radiators 
+
+Historically these days have been internally facing, with someone pitching an idea and just our team in attendance. This month we decided to try an alternate format, we used the time allocated to attend a public hack day called Homeless Hack. 
+
+I thought I'd share some of the thinking behind this and take you through what we created at the hack. Hopefully this will inspire some other organisations to follow suit and use hack days for good!
+
+#### Default to open
+We have always been an open company and publishing our [company handbook](https://github.com/madetech/handbook), [frequently blogging](https://www.madetech.com/blog), [mentoring new programmers](https://github.com/madetech/handbook/blob/master/roles/engineer.md#mentor) or [writing books](https://www.amazon.co.uk/Building-High-Performance-Agile-Teams/dp/1544972687) were steps towards being even more open. It felt at odds with our culture to run hack days that were closed events. Homeless Hack was a good opportunity to rectify this. 
+
+#### Be a socially responsible SME
+Historically we've never given much thought to things like Corporate Social Responsibility. We're an SME, we've got less than 30 people working for us and we like to think we run the business ethically and fairly. 
+
+It's always felt like the world of CSR didn't really apply to us. e.g. It's for the big enterprises, who do bad things and need to offset the bad things by doing some good things. This was a good opportunity to course correct some of our thinking around this and invest some time and money into something that would do some social good. 
+
+#### What are you doing on Saturday?
+We asked everyone in the company to attend HomelessHack, though had just 7 people attend. The event fell on a Friday and Saturday and many people couldn't give up a day of their weekend to attend. Interestingly, we had quite a few people who originally said they would attend, but as the event got closer, had to drop out.
```

Comment:
Maybe something like:

We invited everybody in the company to attend HomelessHack, though as the event neared, the number whittled down to 7.